src/components/fields/Field.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import MoneyInput from './Money';
+import TextField from '@material-ui/core/TextField';
+
+
+export default (name, type, label, obj) => {
```

Comment:
let's change this to a React Component instead of a function so we can implement it as JSX: `<Input config={config}/>`